src/cosmosdb-continuous-backup/azext_cosmosdb_pitr/vendored_sdks/azure_mgmt_cosmosdb/models/api_properties_py3.py

```diff
@@ -0,0 +1,29 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from msrest.serialization import Model
+
+
+class ApiProperties(Model):
+    """ApiProperties.
```

Comment:
Why are all the files duplicated ? there exist *_py3.py for each file. 
Look here: [https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-mgmt-cosmosdb/azure/mgmt/cosmosdb/operations](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-mgmt-cosmosdb/azure/mgmt/cosmosdb/operations)
This is not expected. 
I would recommend downloading the sdk generated with the PR in azure-rest-api-specs (instead of generating the sdk on our own)

Reply:
Added notes similar to readme

Reply:
The sdk files were not generated with the swagger PR. 
Since Vinh did not update the readme file, the python sdk generated was based out of 2020-09-01 version. So we do not have a sdk that is generated on the 2020-06-01-preview version,

Reply:
Shall I ask Vinh to request the release of the python sdk for 2020-06-01-preview so that we can get the sdk that is not locally generated?